openwisp_monitoring/check/tasks.py

```diff
@@ -54,3 +54,32 @@ def auto_create_ping(model, app_label, object_id, created):
     check = Check(name='Ping', check=ping_path, content_type=ct, object_id=object_id)
     check.full_clean()
     check.save()
+
+
+@shared_task
+def auto_create_config_modified(model, app_label, object_id):
+    """
+    Called by openwisp_monitoring.check.models.auto_config_modified_receiver
+    """
+    Check = load_model('check', 'Check')
+    config_modified_path = 'openwisp_monitoring.check.classes.ConfigModified'
+    has_check = (
+        Check.objects.filter(
+            object_id=object_id,
+            content_type__model='device',
+            check=config_modified_path,
+        ).count()
```

Comment:
Tip: there is `.exists()` which replaces `.count() > 0`

Reply:
I do know :sweat_smile:. I thought of using it but then dropped the idea to be consistent with `ping`'s code. Thanks, I shall change for both the checks, then :)